src/pages/PlanPage/subRoutes/CurrentOrgPlan/CurrentOrgPlan.tsx

```diff
@@ -20,10 +20,10 @@ function CurrentOrgPlan() {
     owner,
   })
 
-  const shouldRenderBillingDetails = [
-    accountDetails?.planProvider !== 'github',
-    !accountDetails?.rootOrganization,
-  ].every(Boolean)
+  const shouldRenderBillingDetails =
+    (accountDetails?.planProvider !== 'github' &&
```

Comment:
I felt this was a little more readable than what we had previously

Reply:
More efficient to no need to create an array